src/main/java/uk/gov/hmcts/reform/jps/services/JohPayrollService.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.hmcts.reform.jps.services;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.jps.domain.JohPayroll;
+import uk.gov.hmcts.reform.jps.repository.JohPayrollRepository;
+
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+@Service
+public class JohPayrollService {
+
+    private final JohPayrollRepository johPayrollRepository;
+
+    public JohPayroll findById(Long johId) {
```

Comment:
Can this be optiona by taking off the orElse?

Reply:
Done